### PR TITLE
#215 put statement support

### PR DESCRIPTION
**Pull request by dehbmarques**

### CODE REVIEW

File: lib/db/statement.js [hunk 1]
```diff
@@ -568,6 +581,20 @@ Statement.prototype.buildStatement = function () {
...
+  if (state.put && state.put.length) {
+    statement.push('PUT');
+    statement.push(state.put.map(function (item) {
+      var objectToAdd = item[1];
+      var keys = Object.keys(objectToAdd);
+      var propertyName = item[0];
+      return keys.map(function (key) {
+        var paramName = 'param' + paramify(propertyName + key) + (this._state.paramIndex++);
+        this.addParam(paramName, objectToAdd[key]);
+        return propertyName + ' = \'' + key + '\', :' + paramName;
```
Comment (phpnode) [inline lib/db/statement.js]: let's use double quotes (not single) around the key name and also use `utils.escape()` on it, just in case.

**Comment by phpnode** [comment]
@dehbmarques thanks for this, looks great, just that one minor nitpick. 

I also wonder if we should use that other syntax for this to avoid known issues:
```sql
UPDATE #20:0 SET embmap.test = "Luca", embmap.test2 = "Alex"
```

**Comment by dehbmarques** [comment]
@phpnode about the syntax, should we create the key using ```PUT``` and set the properties using ```SET```?

**Comment by phpnode** [comment]
I think it's good as it is for now, thanks!

**Comment by dehbmarques** [comment]
ok, thanks!

**Pull request merged**